specification/columns/capacityreservationtotal.md

```diff
@@ -0,0 +1,32 @@
+# Capacity Reservation Total
```

Comment:
When would this be different from ConsumedQuantity? If we do need it, I would name it "CapacityReservationQuantity" to be consistent with other quantity column names.